content/general/concepts/cicd-pipeline/cicd-pipeline.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'CICD Pipeline'
+Description: 'CICD Pipeline automates the software / application delivery process.'
+
+Subjects: 
+  - 'CI'
+  - 'CD'
+  - 'The Other CD'
+  - 'Steps in the Pipeline'
+Tags:
+  - 'CICD'
+  - 'Automation'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**CICD Pipeline** is the combination of continuous integration, delivery and deployment of software development and application deployment. The four main steps of the pipeline are: Source, Build, Test and Deploy. 
+
+## CI
+
+CI stands for Continuous Integration. This is the automation process for Developers. 
+
+## CD
+
+The first CD stands for Continuous Delivery. This is the automated delivery of code to the testing and development steps of the pipeline. These environments are usually in a repository like [Github](https://www.github.com)
+
+## The other CD
+
+The second CD stands for Continuous Deployment. When code passes the automated tests, it is automatically placed in production, which is then used for automatic deployment. 
+
+## Steps of the CICD Pipeline
+
+There are four main steps to the CICD Pipeline: 
+    1. Source
+    2. Build
+    3. Test
+    4. Deploy
+
+
+ 
+
+![CICD Pipeline](https://www.redhat.com/themes/custom/rhdc/templates/layout/optimize-test/img/ci-cd-flow-desktop.webp)
+
+
+
```

Comment:
Remove all the extra line breaks at the end of the entry.

Reply:
will do. Thank you for all the edits. I will make the changes and ask for another look!